ddtrace/tracer.py

```diff
@@ -73,15 +77,20 @@ def configure(self, enabled=None, hostname=None, port=None, sampler=None):
         if sampler is not None:
             self.sampler = sampler
 
-    def trace(self, name, service=None, resource=None, span_type=None):
-        """Return a span that will trace an operation called `name`.
+    def trace(self, name, service=None, resource=None, span_type=None, ctx=None, span_parent=None):
```

Comment:
We will have to be very rigorous about our documentation her, since it won't be clear when `span_parent` must be used vs. when `ctx` will be enough.
Also, we could technically have `span_parent` only (`ctx` can come from it). But I'm not sure that is the nicerst API.
What is OT doing here?